src/stores/document-lock-store.js

```diff
@@ -1,11 +1,13 @@
 import Database from '../stores/database.js';
 import LockStoreBase from './lock-store-base.js';
 
+const LOCK_EXPIRATION_TIME_SPAN = { minutes: 30 };
```

Comment:
I was thinking if we need such a long expiration. Maybe 1 minute is enough?

Reply:
That was the next step, consulting you on how long this should be 🤔 I also think 1min should be enough, maybe even less but let's air on the safe side.

Reply:
Reduced to 1 min for document, batch and room locks.